spec/features/roles_spec.rb

```diff
@@ -0,0 +1,34 @@
+require 'rails_helper'
+
+describe 'assigning roles' do
+  context 'with admin privilege' do
+    it 'assigns one role to user' do
+      admin = create :user, admin: true
+      login_as admin
```

Comment:
sign_in is used by devise and can't be used in feature tests. However, login_as is used by warden which can be used in feature tests.